src/http/validations/device.js

```diff
@@ -0,0 +1,18 @@
+import FieldErrors from './fieldErrors';
+
+const getSingleDevice = (req, res, next) => {
+  const { deviceId } = req.params;
+
+  const fieldErrors = new FieldErrors();
+
+  if (!deviceId || typeof (deviceId) !== 'string') fieldErrors.addError('deviceId', 'deviceId is a required string');
+
+  if (fieldErrors.count > 0) {
+    return res.status(400).json({ success: false, message: 'Invalid request body', errors: fieldErrors.errors });
```

Comment:
`deviceId` is not in the request body so I think a message like this wold be better:
```suggestion
    return res.status(400).json({ success: false, message: 'Invalid request', errors: fieldErrors.errors });
```

Reply:
fixed